src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetStandardsListRequest.cs

```diff
@@ -15,6 +14,7 @@ public class GetStandardsListRequest : IGetApiRequest
         public string GetUrl => BuildUrl();
 
         public List<int> Levels { get ; set ; }
+        public OrderBy OrderBy { get; set; }
```

Comment:
won't you need that in build url as well?

Reply:
Updated to be in the request to the inner api